### PR TITLE
David/acs 0230 private

### PR DESCRIPTION
**Pull request by DavidZeLiang**
# Choose a PR Template

Switch to "Preview" on this description then select one of the choices below.

<a href="?expand=1&template=data_plane_template.md">Click here</a> to open a PR for a Data Plane API.

<a href="?expand=1&template=control_plane_template.md">Click here</a> to open a PR for a Control Plane (ARM) API.

### CODE REVIEW

File: specification/cognitiveservices/ContentSafety/models.tsp [hunk 1]
```diff
@@ -228,3 +256,63 @@ model JailbreakAnalysisResult {
...
+
+  @doc("Last build update time.")
+  lastBuildUpdatedTime: utcDateTime;
+}
+
+#suppress "@azure-tools/typespec-azure-core/documentation-required" "MUST fix in next update"
+@doc("Status of the customized category build.")
+enum CategoryBuildStatus {
+  NotStarted,
+  Building,
```
Comment (mengaims) [inline specification/cognitiveservices/ContentSafety/models.tsp:310]: I think "Running" is better.

File: specification/cognitiveservices/ContentSafety/models.tsp [hunk 2]
```diff
@@ -228,3 +256,63 @@ model JailbreakAnalysisResult {
...
+  @visibility("read")
+  version: int32;
+
+  @doc("Creation time of the customized category.")
+  @visibility("read")
+  createdTime: utcDateTime;
+
+  @doc("Build information for the customized category.")
+  @visibility("read")
+  buildInfo: TextCustomizedCategoryBuildInfo;
```
Comment (mengaims) [inline specification/cognitiveservices/ContentSafety/models.tsp:291]: I suggest to put status out, and you can host lastBuildTriggeringTime, lastBuiltUpdatedTime, and "buildErrors" in the buildInfo.

File: specification/cognitiveservices/ContentSafety/models.tsp [hunk 3]
```diff
@@ -228,3 +256,63 @@ model JailbreakAnalysisResult {
...
+}
+
+@doc("Detailed information of the customized category build.")
+model TextCustomizedCategoryBuildInfo {
```
Comment (mengaims) [inline specification/cognitiveservices/ContentSafety/models.tsp:295]: I think we would need errors here if we remove operations.

File: specification/cognitiveservices/ContentSafety/models.tsp (hunk 2 above)
Comment (DavidZeLiang) [reply specification/cognitiveservices/ContentSafety/models.tsp:295]: d

**Pull request merged**